server/src/main/java/fr/uga/l3miage/tp4/models/CandidateEntity.java

```diff
@@ -0,0 +1,21 @@
+package fr.uga.l3miage.tp4.models;
+
+import javax.persistence.*;
+import java.time.LocalDate;
+import java.util.Set;
+
+@Entity
+public class CandidateEntity extends UserEntity{
+    @Id
+    private LocalDate birthDate;
+    private boolean hasExtraTime;
+
+    @ManyToOne
+    @JoinColumn(name = "test_center_id")
+    private TestCenterEntity testCenterEntities;
+
+    @OneToMany
+    @JoinColumn(name = "sheet_number", referencedColumnName = "id")
+    private Set<CandidateEvaluationGridEntity> candidateEvaluationGridEntities;
```

Comment:
ici il fallait utiliser le mappedBy
```suggestion
    @OneToMany(mappedBy="candidateEntity")
    private Set<CandidateEvaluationGridEntity> candidateEvaluationGridEntities;
```